### DISCUSSION TITLE
E: Unable to locate package kali-desktop-xfce-y

### DISCUSSION THREAD

**Discussion by EvanThomasKaduthanam** [microsoft/WSL#7648] [Q&A]
Hello Guys, My Name is Evan and I was trying to download Kali Linux through WSL by following Network Chuck's https://www.youtube.com/watch?v=AfVH54edAHU video... 

So, I was able to download the Kali Kernel and Run it successfully, but not able to download the GUI.

When I ran: `sudo apt update && sudo apt upgrade -y` 
I went kinda well, but It showed: `ldconfig: /usr/lib/wsl/lib/libcuda.so.1 is not a symbolic link`.

I didn't mind it much but then, when I tried to install kali dekstop via `sudo apt install kali-desktop-xfce-y`, 
It is showing: `E: Unable to locate package kali-desktop-xfce-y`

I am a 13 year old student in India and I got interested in Kali Linux and I am really new(only 1 hour into Kali) to this. Anyone's got a fix for this?

**Comment by kabyle24** [comment]
Same to me so please a solution !

**Reply by zidanio17** [reply]
try "sudo apt install kali-desktop-xfce"

**Reply by Zorzi3** [reply]
> try "sudo apt install kali-desktop-xfce"

It doesn't work even with this

**Reply by dannisking** [reply]
> try "sudo apt install kali-desktop-xfce"

this worked for me. I had the same issue

**Reply by Cracker099** [reply]
pour moi aussi cela ne fonctionne pas

**Comment by Darkmintis** [comment]


**Reply by versenilvis** [reply]
> same problem it has been 7 months and still finding its solution. Anybody please help!

Open `etc/apt/sources.list `

then remove all
Add this:

```
deb http://http.kali.org/kali kali-rolling main contrib non-free 
deb-src http://http.kali.org/kali kali-rolling main contrib non-free 
```

then `apt-get update && apt-get upgrade -y `

hope it works

AND TAKE NOTE: 
It is: `sudo apt install -y kali-desktop-xfce` or `sudo apt install kali-desktop-xfce -y`
Not: `sudo apt install kali-desktop-xfce-y` or `sudo apt install kali-desktop - xfce -y`

**Comment by hugomasson55** [comment]
THANK YOU
